src/components/ui/Pool/PoolDeposit.tsx

```
@@ -110,54 +122,196 @@ const PoolDeposit = ({
+        </Button>
+        <Button className="bg_turq" onClick={() => setSliderValue(100)}>
+          <Typography secondary size="xs" fw={700} tt="uppercase">
+            MAX
+          </Typography>
+        </Button>
+      </div>
+      <Button
+        onClick={
+          isConnected
+            ? () => onZapIn(zapAmount, poolTokens[zapTokenIndex].address)
+            : () => openConnectModal && openConnectModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)